client/java-spring-boot-autoconfigure/build.gradle

```diff
@@ -1,7 +1,8 @@
 dependencies {
     compile project(':client:java-armeria-legacy')
-    compile 'javax.validation:validation-api'
+    compileOnly 'javax.validation:validation-api'
```

Comment:
Looks `centraldogma` uses gradle `5.2.1`.
Then, how about `api` instead of `compileOnly`?

https://docs.gradle.org/current/userguide/java_library_plugin.html

Reply:
line/gradle-scripts does not support the new configuration model yet, so we have to stick to the old model.